fragments/header.sh

```diff
@@ -157,6 +157,10 @@ DIALOG_LIST_ITEM_NAME=""
 # listitem.
 # When the variable is unset, progress will be sent to Swift Dialog's main progress bar.
 
+# Add installed App to the end of existing user's using dockutil
+ADDTODOCK=""
```

Comment:
Standard format for user-settable variables in Installomator is capitalized snake-case, so please change this to `ADD_TO_DOCK`